docs/ai-developers/tokens.md

```diff
@@ -24,21 +24,21 @@ However, to understand the different network tokens, you can follow principles l
 * Mainnet tokens could have a monetary value.
 * Both tokens can be used to add extra utility to the software.
 
-We only use Kovan and Ropsten Testnet AGI and the current beta stage. 
+We only use Kovan and Ropsten Testnet AGIX and the current beta stage. 
 
-To learn more about  Mainnet AGI Token (to be used in a later stage) refer to this link [here](https://etherscan.io/address/0x8eb24319393716668d768dcec29356ae9cffe285) with the address `0x8eb24319393716668d768dcec29356ae9cffe285`.
```

Comment:
Change the address

Reply:
@anandrgitnirman  Address change here also